_includes/pbc/all/install-features/202304.0/install-the-inventory-management-feature.md

```diff
@@ -0,0 +1,762 @@
+| src/Orm/Zed/Stock/Persistence/Base/SpyStockStoreQuery.php                        | Spryker/Zed/Stock/Persistence/Propel/AbstractSpyStockStoreQuery.php                        |
+| src/Orm/Zed/StockAddress/Persistence/Base/SpyStockAddress.php                    | Spryker/Zed/StockAddress/Persistence/Propel/AbstractSpyStockAddress.php                    |
+| src/Orm/Zed/StockAddress/Persistence/Base/SpyStockAddressQuery.php               | Spryker/Zed/StockAddress/Persistence/Propel/AbstractSpyStockAddressQuery.php               |
+| src/Orm/Zed/WarehouseAllocation/Persistence/Base/SpyWarehouseAllocation.php      | Spryker/Zed/WarehouseAllocation/Persistence/Propel/AbstractSpyWarehouseAllocation.php      |
+| src/Orm/Zed/WarehouseAllocation/Persistence/Base/SpyWarehouseAllocationQuery.php | Spryker/Zed/WarehouseAllocation/Persistence/Propel/AbstractSpyWarehouseAllocationQuery.php |
+
+{% endinfo_block %}
+
+
+{% info_block warningBox "Verification" %}
+
+Make sure that `SpyAvailabilityTableMap::getBehaviors()` provides mapping for `spy_availability_is_never_out_of_stock`, `spy_availability_quantity`, and `spy_availability_sku`.
+
+{% endinfo_block %}
+
+### 3) Configure OMS.
+
+Create the OMS sub-process file.
+
+**config/Zed/oms/WarehouseAllocationSubprocess/WarehouseAllocation01.xml**
+
+```xml
+<?xml version="1.0"?>
+<statemachine
+        xmlns="spryker:oms-01"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="spryker:oms-01 http://static.spryker.com/oms-01.xsd"
+>
+
+    <process name="WarehouseAllocation">
+        <states>
+            <state name="warehouse allocated" reserved="true"/>
+        </states>
+
+        <events>
+            <event name="allocate warehouse" onEnter="true" command="WarehouseAllocation/WarehouseAllocate"/>
+        </events>
+    </process>
+</statemachine>
+```
+
+Using the `DummyPayment01.xml` process as an example, adjust your OMS state-machine configuration according to your project’s requirements.
+
+<details open>
+    <summary markdown='span'>config/Zed/oms/DummyPayment01.xml</summary>
+
+```xml
+<?xml version="1.0"?>
+<statemachine
+        xmlns="spryker:oms-01"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="spryker:oms-01 http://static.spryker.com/oms-01.xsd"
+>
+    <process name="DummyPayment01" main="true">
+        <transitions>
+            
+            <transition happy="true">
+                <source>new</source>
+                <target>warehouse allocated</target>
+                <event>allocate warehouse</event>
+            </transition>
+    
+        </transitions>
+        
+        <subprocesses>
+            <process>WarehouseAllocation</process>
+        </subprocesses>
+        
+    </process>
+    
+    <process name="WarehouseAllocation" file="WarehouseAllocationSubprocess/WarehouseAllocation01.xml"/>
+
+</statemachine>
+```
+
+
+Enable the following plugins.
```

Comment:
Move it to `Set up behavior` section.

Reply:
You shouldn't create new `Set up behavior` section, just move plugin to already existing `Set up behavior`.